pkg/resource/feature_group/custom_update_api_test.go

```diff
@@ -0,0 +1,41 @@
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package feature_group
+
+import (
+	mocksvcsdkapi "github.com/aws-controllers-k8s/sagemaker-controller/mocks/aws-sdk-go/sagemaker"
+	ctrlrtzap "sigs.k8s.io/controller-runtime/pkg/log/zap"
+	ackmetrics "github.com/aws-controllers-k8s/runtime/pkg/metrics"
+	"go.uber.org/zap/zapcore"
+)
+
+// provideResourceManagerWithMockSDKAPI accepts MockSageMakerAPI and returns pointer to resourceManager
+// the returned resourceManager is configured to use mockapi api.
+func provideResourceManagerWithMockSDKAPI(mockSageMakerAPI *mocksvcsdkapi.SageMakerAPI) *resourceManager {
```

Comment:
why is this file named `custom_update_api_test`

this was part of manager_test_suite.go in other PR

Reply:
This change was added in PRs #66 and #76. It has now been added to this one as well.